targets/esp8266/readme.md

```diff
@@ -56,26 +36,37 @@ Default USB device is `/dev/ttyUSB0`. If you have different one, give with `USBD
 USBDEVICE=/dev/ttyUSB1 make -f ./targets/esp8266/Makefile.esp8266 flash
 ```
 
+### 4. Running
+
+* power off
+* connect GPIO2 with serial of 470 Ohm + LED and to GND
+* power On
+
+LED should blink on and off every second
+
+#### 5. Cleaning
 
-### 5. Running
+To clean the build result:
 
-1. Power off
-2. Disonnect(float) both GPIO0 and GPIO2
-3. Power on
+```
+make -f ./targets/esp8266/Makefile.esp8266 clean
+```
 
-Sample program here works with LED and a SW with below connection.
-* Connect GPIO2 to a LED > 4K resistor > GND
-* Connect GPIO0 between VCC > 4K resistor and GND
+If the board already been flashed it may contains written areas which will not be overwritten by flashing jerryscript.
```

Comment:
this sentence needs a bit more massaging. "has already been flashed", "may contain": I'm almost sure that these changes are needed. but I'm not sure what the "written areas which will not be overwritten by flashing" means.

Reply:
I'll give more details what "written areas which will not be overwritten by flashing" means

Reply:
Let me correct myself, `erase_flash` rule is optional and doesn't need any prerequisites. So this description is unnecessary.